features/step_definitions/deletion_steps.rb

```diff
@@ -0,0 +1,37 @@
+Then(/^I should not see a link to delete the post$/) do
+  expect(page).not_to have_link("Destroy")
+end
+
+Given(/^I am a registered user$/) do
+  @me = FactoryGirl.create(:user,
+                           username: "norm",
+                           email: "norm@example.com",
+                           password: "12345678")
+end
+Given(/^there is a post of mine to view$/) do
+  @post = FactoryGirl.create(:post, user_id: @me.id)
+end
+
+Given(/^there is someone elses post to view$/) do
+  @otherUser = FactoryGirl.create(:user,
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)